packages/pluggable-widgets-tools/test-config/jest.config.js

```diff
@@ -4,28 +4,25 @@ const projectDir = process.cwd();
 
 module.exports = {
     clearMocks: true,
+    testRunner: "jest-jasmine2",
     rootDir: join(projectDir, "src"),
     setupFilesAfterEnv: [join(__dirname, "test-index.js")],
     snapshotSerializers: ["enzyme-to-json/serializer"],
+    testEnvironment: "jsdom",
     testMatch: ["<rootDir>/**/*.spec.{js,jsx,ts,tsx}"],
     transform: {
-        "\\.tsx?$": [
-            "ts-jest",
-            {
-                isolatedModules: true,
-                tsconfig: { module: "commonjs" }
-            }
-        ],
-        "\\.jsx?$": join(__dirname, "transform.js"),
-        "^.+\\.svg$": "jest-svg-transformer"
+        "^.+\\.tsx?$": ["ts-jest", { tsconfig: { module: "commonjs" } }],
```

Comment:
It is rolling back the `isolatedModules: true,` ts-jest config.
We may have to disable type checkings to improve the pwt commands execution time

Reply:
Restored